### ISSUE TITLE
Question: Is it possible to use extended_name when exporting the configuration?

### ISSUE THREAD

**Issue by jiachenwang** [airbnb/lottie-web#2630]
Currently exported configuration uses short json name. For example:
  - In [valueKeyframed.json](https://github.com/airbnb/lottie-web/blob/master/docs/json/properties/valueKeyframed.json), `k` is for Keyframes.
  - In [value.json](https://github.com/airbnb/lottie-web/blob/master/docs/json/properties/value.json), `k` is for Value.

This causes issues. When Opacity field has both `#/properties/value` and `#/properties/valueKeyframed` in the `oneOf`, exported json only has the short name `k`:
```
{
    "o":
    {
        "a": 0,
        "k": 100,
        "ix": 11
    }
}
```
This causes problem when converting this json configuration to protobuf since the json name `k` here is ambiguous.

Basically, is it possible to let Lotti or Bodymovin use extended_name when exporting the json configuration? Thanks very much in advance!
